src/components/ChatLog.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ({entries, updateLikes}) => {
+    const entryComponents = [];
+    // This is the same as const entries = props.entries
+
+    for (const message of entries) {
+        entryComponents.push(
+            <ChatEntry 
+                key={message.id}
+                id={message.id}
+                sender={message.sender} 
+                body={message.body} 
+                timeStamp={message.timeStamp} 
+                liked={message.liked}
+                updateLikes={updateLikes}
+            />
+        );
+    }
+    return <div className='chat-log'>{entryComponents}</div>;
+};
```

Comment:
We could also use the `map` function as it builds a list as it iterates through `entries` (another list). 

```javascript
    let entryComponents = entries.map((message) => {
            <ChatEntry 
                key={message.id}
                id={message.id}
                sender={message.sender} 
                body={message.body} 
                timeStamp={message.timeStamp} 
                liked={message.liked}
                updateLikes={updateLikes}
            />
        };
    )
    return <div className='chat-log'>{entryComponents}</div>;
};
```